linker/Tests/Mono.Linker.Tests.Cases/TypeForwarding/MissingTargetReference.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using Mono.Linker.Tests.Cases.Expectations.Assertions;
+using Mono.Linker.Tests.Cases.Expectations.Metadata;
+
+namespace Mono.Linker.Tests.Cases.TypeForwarding
+{
+	[Define ("IL_ASSEMBLY_AVAILABLE")]
+	[SetupCompileBefore ("TypeForwarderMissingReference.dll", new [] { "Dependencies/TypeForwarderMissingReference.il" })]
+	[SetupLinker ("-p link TypeForwarderMissingReference.dll")]
```

Comment:
What do you think of going with `SetupLinkerAttribute(string flag, params string[] values)` and accepting multiple attributes, instead of a single string?  There are a few reasons why I think that will work better.

* You added a call to .Split(' ') in the test framework code, that's not going to work correct if there's ever a space in a file path (admittedly a low chance).  But in generally I'd prefer to avoid getting the test framework involved in linker argument parsing.

* Our UnityLinker doesn't use the same command line argument parsing logic as upstream.  The upstream logic is quite bare bones.  Instead we use our own options parsing logic, which is itself based off https://github.com/mono/mono/blob/master/mcs/class/Mono.Options/Mono.Options/Options.cs.  Anyways, long story short, our testing infrastructure for the UnityLinker needs to translate these arguments to the equivalent for our linker.  I think translation will be easier & less brittle if we minimize the amount of string parsing needed. 

* By separating out the flag into it's on parameter, it will make it easier for us to translate the arguments.  The idea would be that there is 1 [SetupLinker] attribute per flag, that way our UnityLinker test framework code doesn't have to scan an array of strings for flags, figure out how many arguments can/should follow in order to translate the arguments.